client/blocks/edit-gravatar/test/index.jsx

```diff
@@ -23,6 +23,10 @@ describe( 'EditGravatar', () => {
 		email_verified: false,
 	};
 
+	const userSettings = {
```

Comment:
This `userSettings` prop can be removed from all the tests now. Most of the test cases are just dealing with situations where `isGravatarProfileHidden` and `isFetchingUserSettings` are falsey, so they're working just fine without the new props having to be explict.